web/src/server/api/routers/traces.ts

```diff
@@ -652,6 +652,15 @@ export const traceRouter = createTRPCRouter({
             projectId: input.projectId,
           },
         }),
+        // given traces and observations live in ClickHouse we cannot enforce a fk relationship and onDelete: Cascade
```

Comment:
kannst du das nach untern schieben in den if (env.CLICKHOUSE_URL) { block? Dann löschen wir das nicht ausversehen, wenn wir die pg logik entferne.

Reply:
@maxdeichmann added both of the relevant comments in the clickhouse block such that this logic is not accidentally deleted, if okay for you feel free to resolve comment and this PR will merge automatically